lib/pure/strformat.nim

```diff
@@ -296,6 +296,32 @@ to be injected into the context where ``myTemplate`` is expanded,
 everything is wrapped in a ``block``.
 
 
+Backslash support
+=================
+
+Nim ``strformat`` supports Backslash ``\``, thanks to triple string literals:
+
+.. code-block:: nim
+
+  echo fmt"""{ "yep\nope" }"""  ## Works Ok.
+
+This is a feature of the triple string literals, not of ``strformat`` itself,
+but they can be used togheter to support the Backslash ``\`` on ``strformat``.
```

Comment:
typo: "together"

Reply:
"the Backslash"? how about "the backslash"

Reply:
I think that section should be removed entirely, it belongs to https://github.com/nim-lang/Nim/wiki/Nim-for-Python-Programmers where it already is :)

Reply:
@juancarlospaco Sorry, I didn't mean the whole PR, just this `Backslash support` section